banking/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I'd move this up one level  to the repo level (it's currently in banking/LICENSE instead of at the top level) and have license headers on files unless there is a plan for different licenses per sample application.

Reply:
LICENSE is already at the root so removing this one, thanks!